raft-ann-bench/gpu/Dockerfile

```
@@ -0,0 +1,43 @@
+# syntax=docker/dockerfile:1
+
+ARG CUDA_VER=12.0.1
+ARG PYTHON_VER=3.10
+ARG LINUX_VER=ubuntu22.04
+
+ARG RAPIDS_VER=23.10
+
+FROM rapidsai/mambaforge-cuda:cuda${CUDA_VER}-base-${LINUX_VER}-py${PYTHON_VER} as raft-ann-bench
+ARG CUDA_VER
+ARG RAPIDS_VER
+
+RUN useradd -rm -d /home/rapids -s /bin/bash -g conda -u 1001 rapids
+
```

Need to switch to the newly created user:
```suggestion

USER rapids

```